lib/opendax/vault.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Opendax
+  class Vault
+    def vault_secrets_path
+      'config/vault-secrets.yml'
+    end
+
+    def exec(command)
+      `docker-compose exec vault sh -c '#{command}'`
```

Comment:
You can use the `sh` method in OpenDAX to execute commands

Reply:
`sh` works only in Rake tasks